fay.cabal

```diff
@@ -149,18 +148,17 @@ library
     , language-ecmascript >= 0.15 && < 0.18
     , mtl >= 2.1 && < 2.3
     , mtl-compat >= 0.1 && < 0.3
-    , process >= 1.1 && < 1.5
+    , process >= 1.1 && < 1.7
     , safe >= 0.2 && < 0.4
     , sourcemap == 0.1.*
     , split >= 0.1 && < 0.3
     , spoon >= 0.1 && < 0.4
     , syb >= 0.3 && < 0.8
     , text >= 0.11 && < 1.3
-    , time >= 1.4 && < 1.7
+    , time >= 1.4 && < 1.9
     , transformers >= 0.3 && < 0.4 || > 0.4.1 && < 0.7
     , transformers-compat >= 0.3 && < 0.6
-    , traverse-with-class >= 0.1 && < 0.3
-    , type-eq >= 0.1 && < 0.6
+    , traverse-with-class >= 1.0 && < 2.0
```

Comment:
To follow the [PVP](https://pvp.haskell.org/) this should be `>= 1.0 && < 1.1` since `1.1` may contain breaking changes.